### ISSUE TITLE
About training set

### ISSUE THREAD

**Issue by xia0min** [he2016012996/CABnet#1]
hi,
I am so appreciated with your great job. And i have read your code carefully.
When i read the train.py, it confused me in
" 'DDR':   [9851,2503,5,'./data/DDR/train','./data/DDR/valid']" 

In your paper,the number of training is 6260,but 9851 in the code.
so how do you get 9851 pictures in the training dataset
look forward to your reply.
thanks.

**Comment by NKUhealong** [comment]
We augment the classes with fewer training samples, such as label 1,label 3 and  label 4, you can  augment these categories

**Comment by xia0min** [comment]
thank you ,i will try it

**Comment by zhangzhuoneng** [comment]
您好，我仔细看了下你的代码，有点疑惑。为什么您的epochs有两个，学习率也有两个呢。那个epoch=1的作用是啥呢？谢谢！

**Comment by houjunlin** [comment]
> 您好，我仔细看了下你的代码，有点疑惑。为什么您的epochs有两个，学习率也有两个呢。那个epoch=1的作用是啥呢？谢谢！

I have the same question. Why set _layer.trainable=False_ and train one epoch?
